azure-pipelines.yml

```diff
@@ -59,7 +59,7 @@ jobs:
   - task: UseDotNet@2
     inputs:
       packageType: 'sdk'
-      version: '2.x'
+      version: '6.x'
```

Comment:
Updated target framework to .NET 6.0 for sample and test projects. .NET Core 2.2 is out of support since two years.